README.md

```diff
@@ -1 +1,151 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [rails Budget app] <a name="about-project"></a>
+
+> In this project you can sign in, create a post, comment on other users posts, and like the posts
+
+**[Ruby on Rails blog app]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">ruby</a></li>
+    <li><a href="https://reactjs.org/">rails</a></li>
+    <li><a href="https://reactjs.org/">postgresql</a></li>
+    <li><a href="https://reactjs.org/">tailwind</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[test]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## live demo
+[live demo linke](https://eldeeb.onrender.com/splash)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1- ruby
+2- rails
+3- postgresql
+4- capybara
+5- mailcatcher (to confirm email in registration)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1- ruby v 3.1.3
+2- rails
+
+### Install
+
+Install this project with:
+
+gem install mailcatcher & refer to the<a href="https://mailcatcher.me/">docs</a> for more instructions
+
+bundle install
+
+### Usage
+
+To run the project, execute the following command:
+
+rails s
+
+### Run tests
+
+To run tests, run the following command:
+
+1- run bundle install
+2- rspec spec
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Mohamed El-Deeb**
+
+- GitHub: [@githubhandle](https://github.com/eng-mohamed-eldeeb)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/eng-mohamed-eldeeb/)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[authorization]**
+- [ ] **[authorization]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing">Peter</a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).